Makefile

```diff
@@ -53,6 +53,15 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+GOLANGCI_LINT = $(GOBIN)/golangci-lint
+# golangci-lint version should be updated periodically
+# we keep it fixed to avoid it from unexpectedly failing on the project
+# in case of a version bump
+GOLANGCI_LINT_VER = v1.46.1
+TIMEOUT = 15
```

Comment:
This variable is actually unused. Maybe you wanted to use it in `-timeout` parameters of `go test`?

Reply:
Removing it.